src/XMakeTasks/UnitTests/GenerateResource_Tests.cs

```diff
@@ -419,7 +417,7 @@ public void ForceSomeOutOfDate()
                 DateTime time2 = File.GetLastWriteTime(t.OutputResources[1].ItemSpec);
 
                 System.Threading.Thread.Sleep(200);
-                if (NativeMethodsShared.IsOSX)
+                if (!NativeMethodsShared.IsWindows)
```

Comment:
Are you seeing this on Linux? I had a similar fix earlier https://github.com/Microsoft/msbuild/commit/0698cc9cffb03edebc4f7d5cd5453f804c97582e . Are you seeing failure in `TrackingDependenciesTests` too?